docs/data/install-hadoop-2-0-0.md

```diff
@@ -0,0 +1,1499 @@
+root# /sbin/service fetch-crl3-cron start
+%RED%# For RHEL 6, CentOS 6, and SL6, or OSG 3 _older_ than 3.1.15 %ENDCOLOR%
+root# /sbin/service fetch-crl-boot start
+root# /sbin/service fetch-crl-cron start
+%RED%# For RHEL 7, CentOS 7, and SL7 %ENDCOLOR%
+root# systemctl start fetch-crl-boot
+root# systemctl start fetch-crl-cron
+```
+
+**NOTE**: while it is necessary to start `fetch-crl-cron` in order to have it active, `fetch-crl-boot` is started automatically at boot time if enabled. The start command will run `fetch-crl-boot` at the moment when it is invoked and it may take some time to complete.
+
+---%SHIFT%++ Configure `fetch-crl` To modify the times that fetch-crl-cron runs, edit `/etc/cron.d/fetch-crl` (or `/etc/cron.d/fetch-crl3` depending on the version you have).
+
+By default, `fetch-crl` connects directly to the remote CA; this is inefficient and potentially harmful if done simultaneously by many nodes (e.g. all the worker nodes of a big cluster). We recommend you provide a HTTP proxy (such as squid) the worker nodes can connect to. [Here](InstallFrontierSquid) are instructions to install a squid proxy.
+
+To configure fetch-crl to use an HTTP proxy server:
+
+-   If using `fetch-crl` version 2 (the `fetch-crl` package on RHEL5 only), then create the file `/etc/sysconfig/fetch-crl` and add the following line: <pre class="file">
+
+export http\_proxy=%RED%http://your.squid.fqdn:port<span class="twiki-macro ENDCOLOR"></span> </pre> Adjust the URL appropriately for your proxy server.
+
+-   If using `fetch-crl` version 3 on RHEL5 via the `fetch-crl3` package or on RHEL6/RHEL7 via the `fetch-crl` package, then create or edit the file `/etc/fetch-crl3.conf` (RHEL5) or `/etc/fetch-crl.conf` (RHEL6/RHEL7) and add the following line: <pre class="file">
+
+http\_proxy=%RED%http://your.squid.fqdn:port<span class="twiki-macro ENDCOLOR"></span> </pre> Again, adjust the URL appropriately for your proxy server.
+
+Note that the **`nosymlinks`** option in the configuration files refers to ignoring links within the certificates directory (e.g. two different names for the same file). It is perfectly fine if the path of the CA certificates directory itself (`infodir`) is a link to a directory.
+
+Any modifications to the configuration file will be preserved during an RPM update.
+
+For more details, please see our [fetch-crl documentation](../common/ca).
+
+Current versions of fetch-crl and fetch-crl3 produce more output. It is possible to send the output to syslog instead of the default email system. To do so:
+
+1.  Change the configuration file to enable syslog: <pre class="file">
+
+logmode = syslog syslogfacility = daemon</pre>
+
+1.  Make sure the file `/var/log/daemon` exists, e.g. touching the file
+2.  Change `/etc/logrotate.d` files to rotate it
+
+
+Installation
+============
+
+Installation depends on the node you are installing:
+
+Namenode Installation
+---------------------
+
+``` console
+root# yum install osg-se-hadoop-namenode
```

Comment:
Could you fix these prompts using this table? https://opensciencegrid.github.io/technology/software/markdown-migration/#fixing-root-and-user-prompts

Sorry, that's my bad due to a bad conversion in the docker image.